ambry-cloud/src/main/java/com/github/ambry/cloud/VcrReplicationManager.java

```diff
@@ -198,16 +193,22 @@ protected int getNumReplThreads(double cpuScale) {
     return (int) (Double.valueOf(Runtime.getRuntime().availableProcessors()) * cpuScale);
   }
 
+  @Override
+  protected void runThread(ReplicaThread replicaThread) {
+    Utils.daemonThread(replicaThread.getName(), replicaThread).start();
+    logger.info("Started replica thread {}", replicaThread.getName());
+  }
```

Comment:
Why do we run vcr replication thread as daemon thread? It doesn't make sense. We should run it as user thread.

Reply:
My idea was to reduce the shutdown time for vcr. If the main thread exits and the others are daemon, then jvm will exit immediately. If this isn't right, i can change it back.

Reply:
Let's not use daemon thread here.